src/lib/jsonParseV2/parseSubscription.cpp

```diff
@@ -100,6 +100,25 @@ std::string parseSubscription(ConnectionInfo* ciP, ParseData* parseDataP, JsonDe
     return oe.render(ciP, "");
   }
 
+  // Description field
+  destination->descriptionProvided = false;
+  if (document.HasMember("description"))
+  {
+    const Value& description      = document["description"];
+    std::string descriptionString = description.GetString();
```

Comment:
Check it is a string before calling GetString or "something bad" (TM) can happen
 (.IsString())